app/services/task_service/gpt_generate_tests.rb

```diff
@@ -2,11 +2,14 @@
 
 module TaskService
   class GptGenerateTests < ServiceBase
-    def initialize(task:)
+    def initialize(task:, user:)
       super()
       raise Gpt::MissingLanguageError if task.programming_language&.language.blank?
 
       @task = task
+      @user = user
+      @access_token = @user.personal_openai_api_key.presence
```

Comment:
Why are we calling the `api_key` `access_token` now?

Reply:
I will try to clean up this area better